.github/workflows/pkgdown.yaml

```diff
@@ -38,15 +38,22 @@ jobs:
           needs: website
 
       - name: Install cmdstan
-        run: |
-          cmdstanr::check_cmdstan_toolchain(fix = TRUE)
-          cmdstanr::install_cmdstan(cores = 2, quiet = TRUE)
-        shell: Rscript {0}
+        uses: epinowcast/actions/install-cmdstan@v1
+        with:
+          cmdstan-version: 'latest'
+          num-cores: 2
 
       - name: Build site
         run: pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE)
         shell: Rscript {0}
 
+      - name: Upload website
```

Comment:
So previously we didn't have this. It renders the website to a folder named `docs` which lives {somewhere} for a period of 5 days?

Reply:
> saving the built website as an enviroment object

So I guess the {somewhere} is an "environment object" (also don't know what this is, looking online now).